rocAL/rocAL_pybind/amd/rali/decoders.py

```diff
@@ -0,0 +1,211 @@
+import amd.rali.types as types
```

Comment:
please change all rali names and folders to rocal

Reply:
Sure Rajy, will make this change in the next incremental Change